src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -213,8 +212,20 @@ protected final String buildMethodName() {
     }
 
     protected final AnnotationSpec generatedAnnotation() {
+        Class<?> generatedAnnotation;
+        try {
+            // available since 9
+            generatedAnnotation = Class.forName("javax.annotation.processing.Generated");
+        } catch (ClassNotFoundException e) {
+            try {
+                generatedAnnotation = Class.forName("javax.annotation.Generated");
+            } catch (ClassNotFoundException ex) {
+                // Shouldn't be possible.
+                throw new IllegalStateException(ex);
+            }
+        }
```

Comment:
Would you mind actually extracting this to a separate helper method? Something like `private Class<?> determineGeneratedAnnotationClass() { ...`?

Reply:
Done